lavalink/Client.py

```diff
@@ -163,29 +181,46 @@ def unregister_hook(self, func):
             return
 
         if data['t'] == 'VOICE_SERVER_UPDATE':
-            self.voice_state.update({
+            voice_server_package = {
                 'op': 'voiceUpdate',
                 'guildId': data['d']['guild_id'],
                 'event': data['d']
-            })
+            }
+            try:
+                self.voice_states[int(data['d']['guild_id'])].update(voice_server_package)

```

Comment:
Why not store the required stuff (I believe this is only `session_id` on a per-player basis?) Might use a little less memory this way

Reply:
If I understand you correctly: when we push the package to Lavalink we remove everything except `session_id`? If that's what you meant sure!

Reply:
```python
player = get_player(guild_id, create=False)

if player:
    player._session_id = new_session_id
```

This assumes that the player exists most, if not all, of the time because one is needed to establish a connection to a voice channel

Reply:
My opinion is to keep those separate, while it doesn't really make sense, but we eliminate some unnecessary iterations to find a player.

Reply:
On the other hand, the issue is not in the session_id, we receive those regularly, but with the `VOICE_SERVER_UPDATE` which has the token and Discord WS.